core/src/worker/data-feed.ts

```diff
@@ -184,6 +185,12 @@ export function aggregatorJob(
     logger.debug(inData, 'inData')
     const { oracleAddress, roundId, workerSource } = inData
 
+    if (!state.isActive({ oracleAddress })) {
+      logger.warn(`aggregatorJob for oracle ${oracleAddress} is no longer active. Removing job.`)
+      job.remove()
```

Comment:
`aggregatorJob` is not repeatable job, therefore it is not necessary to "remove" it like this. I am not sure it would even have any effect if it is repeatable job anyway. Since the job is being recreated independently.

```suggestion
```